src/extlibs/hrvo/simulator.h

```diff
@@ -94,38 +94,43 @@ class HRVOSimulator
     /**
      *      Adds a new agent to the simulation.
      *
-     * @param position           The starting position of this agent.
-     * @param goal_index          The index of the Goal which this agent should go to.
-     * @param neighborDist       The maximum neighbor distance of this agent.
-     * @param maxNeighbors       The maximum neighbor count of this agent.
-     * @param agent_radius       The agent_radius of this agent.
-     * @param goalRadius         The goal agent_radius of this agent.
-     * @param prefSpeed          The preferred speed of this agent.
-     * @param maxSpeed           The maximum speed of this agent.
-     * @param uncertaintyOffset  The uncertainty offset of this agent.
-     * @param maxAccel           The maximum acceleration of this agent.
-     * @param curr_velocity      The initial velocity of this agent.
+     * @param position              The starting position of this agent.
+     * @param goal_index            The index of the Goal which this agent should go to.
+     * @param neighborDist          The maximum neighbor distance of this agent.
+     * @param maxNeighbors          The maximum neighbor count of this agent.
+     * @param agent_radius          The agent_radius of this agent.
+     * @param max_radius_inflation  The maximum amount which the radius of this agent can
+     * inflate.
+     * @param goalRadius            The goal agent_radius of this agent.
```

Comment:
what is goal radius?

Reply:
Its an old parameter which is passed as a part of the path. It basically represents how close to the path point you need to be in order to set the destination as the next path point. 
I've removed this comment. Good catch.